ios/ExpensifyCash.xcodeproj/project.pbxproj

```diff
@@ -328,6 +328,10 @@
 			buildActionMask = 2147483647;
 			files = (
 			);
+			inputPaths = (
+				"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}",
+				"$(SRCROOT)/$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)",
+			);
```

Comment:
It's not clear to me why these changes are being made as well as the podfile changes below.

Reply:
I think these were auto-generated during pod install .